cmake/common.cmake

```diff
@@ -358,24 +378,28 @@ if("sgx" IN_LIST COMPILE_TARGETS)
     EXPORT ccf
     DESTINATION lib
   )
+  list(APPEND JS_PLUGINS_ENCLAVE js_openenclave.enclave)
 endif()
-if("virtual" IN_LIST COMPILE_TARGETS)
-  add_library(js_openenclave.virtual STATIC ${CCF_DIR}/src/js/openenclave.cpp)
-  add_san(js_openenclave.virtual)
-  target_link_libraries(js_openenclave.virtual PUBLIC ccf.virtual)
-  target_compile_options(js_openenclave.virtual PRIVATE ${COMPILE_LIBCXX})
-  target_compile_definitions(
-    js_openenclave.virtual PUBLIC INSIDE_ENCLAVE VIRTUAL_ENCLAVE
-                                  _LIBCPP_HAS_THREAD_API_PTHREAD
-  )
-  set_property(
-    TARGET js_openenclave.virtual PROPERTY POSITION_INDEPENDENT_CODE ON
-  )
-  install(
-    TARGETS js_openenclave.virtual
-    EXPORT ccf
-    DESTINATION lib
-  )
+if("virtual" IN_LIST COMPILE_TARGETS AND NOT DISABLE_OE)
+  if (NOT DISABLE_OE)
```

Comment:
redundant with check above?

Reply:
Yes, I was oscillating between the two and ended up forgetting to remove one. Good catch!